lib/luafib.c

```diff
@@ -60,20 +65,30 @@ static int luafib_nl_rule(luafib_rule_t *rule)
 	nlmsg_end(skb, nlh);
 
 	skb->sk = rule->net->rtnl;
+	#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 15, 0)
+	ret = rule->command(rule->net, skb, nlh, NULL, true);
+	#else
 	ret = rule->command(skb, nlh, NULL);
+	#endif
 free:
 	nlmsg_free(skb);
 out:
 	return ret;
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 15, 0)
+#define LUAFIB_RULECOMMAND(op) fib_##op
+#else
+#define LUAFIB_RULECOMMAND(op) fib_nl_##op
+#endif
```

Comment:
```suggestion
#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 15, 0)
#define LUAFIB_PREFIX		fib
#else
#define LUAFIB_PREFIX		fib_nl
#endif
```

it's completely up to you.. but I think this version is simpler ;-)

Reply:
I tried, but then I can’t get to find the right syntax to concatenate it with `op` within `#define LUAFIB_OPRULE(op)`.